week-3/quote-generator/style.css

```diff
@@ -1 +1,29 @@
 /** Write your CSS in here **/
+body {
+    font-family: Arial, sans-serif;
+    background-color: #f1f7ff;
+    margin: 25% 10%;
+    padding: 5%;
+
+}
+
+h1 {
+    justify-content: center;
+    color: rgb(160, 38, 38);
+}
+
+button {
+    padding: 5px 10px;
+    background-color: rgb(209, 166, 141);
+    border-radius: 10px;
+}
+
+p {
+    justify-content: center;
```

Comment:
I don't think this `justify-content` property is doing what you expect, as it's only relevant when using Flexbox. Are you possibly looking for `text-align`?

Reply:
I believe you are right, I was likely looking for that